Symfony/CS/Fixer/Symfony/PhpdocScalarFixer.php

```diff
@@ -34,13 +34,53 @@ class PhpdocScalarFixer extends AbstractFixer
      * @var array
      */
     private static $types = array(
-        'integer' => 'int',
         'boolean' => 'bool',
-        'real' => 'float',
         'double' => 'float',
+        'integer' => 'int',
+        'real' => 'float',
+    );
+
+    /**
+     * The supported types to process.
+     *
+     * @var array
+     */
+    private static $supported = array(
+        'array',
```

Comment:
Yeh, I totally agree. I was thinking that when I was adding them. Maybe we need to call it phpdoc_type_normalizer or something?